src/main/java/com/dev/cinema/PasswordValidator.java

```diff
@@ -0,0 +1,28 @@
+package com.dev.cinema;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = PasswordValidatorImpl.class)
+@Target({ ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface PasswordValidator {
+    String fieldMatch();
+
+    String message();
+
+    String field();
+
+    Class<?>[] groups() default {};
+    Class<? extends Payload>[] payload() default {};
+
+    @Target({ ElementType.TYPE })
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface List {
+        PasswordValidator[] value();
+    }
```

Comment:
do we need this?

Reply:
no, but yesterday i thought that it was a need